sprint02/task12/js/script.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+
+
+function concat(str1, str2) {
+
+	function innerConcat(str2) {
```

Comment:
It's important to avoid naming parameters in the inner function the same as they are named in the parent function. This can cause confusion and unexpected behavior. In this case, it's recommended to change the name of `str2` to something more descriptive to avoid any potential issues.